tox.ini

```diff
@@ -0,0 +1,15 @@
+[tox]
+requires =
+    tox-poetry-installer[poetry]==1.0.0b1
+
+envlist = py37, py38, py310
+isolated_build = true
+
+[testenv]
+require_poetry = true
+require_locked_deps = true
+poetry_dep_groups =
+    dev
+commands =
+    poe generate
```

Comment:
I don't think we should be making this part of the `tox` commands - `poe generate` can be quite slow. Also, is there a reason why this is now a separate file? Probably best not to make it a separate file if we can.